### DISCUSSION TITLE
Installing XP 64-bit

### DISCUSSION THREAD

**Discussion by hydrohs** [ipxe/ipxe#1037] [Q&A] [answered]
Is it possible to install XP 64-bit via iPXE? I haven’t been able to find anything specific, and trying to adapt the methods for 32-bit have been a no-go.

**Comment by NiKiZe** [comment, marked as answer]
Should be same as 32, XPs first part of setup runs in pure BIOS mode, then drivers is loaded and real time kernel kicks in.
In PXE mode you need the services to provide all needed files and configurations. Ther eshould be no difference between iPXE and PXE here.

But as always when asking for help, provide the details of what you have, and where you get stuck with what messages, and what you have tried and if that yielded any and what change.

**Comment by hydrohs** [comment]
You are totally right. I made a dumb mistake. I was trying to use my SP2 install disc and not the actual OS install disc. All works as expected, appreciate the help.
